sklearn/datasets/base.py

```diff
@@ -255,14 +255,22 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+     ----------
+    return_X_y : boolean, default=False.
+        If True, returns (data, target) instead of a Bunch object.
+        See below for more information about the `data` and `target` object.
+
     Returns
     -------
     data : Bunch
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
-        full description of the dataset.
+        full description of the dataset. This is test text.
```

Comment:
Yep. That's what I'm doing. 

![image](https://cloud.githubusercontent.com/assets/12952080/17032804/b54a2d3a-4f98-11e6-823f-1299516cbf59.png)


![image](https://cloud.githubusercontent.com/assets/12952080/17032777/94516238-4f98-11e6-9055-e07f9ee0b2ed.png)